src/content/datasheets/photon-shields.md

```diff
@@ -23,14 +23,17 @@ We use Texas Instruments TXB0108PWR to do the voltage translation in between the
 
 The new version of the Shield Shield (v3.x.x) uses dedicated mosfet based voltage translation on the I2C lines. We also decided to add a prototyping area in empty space in the middle of the shield.
 
+#### Notes
+
+1. One drawback of using the TXB0108PWR as a voltage translator is that it is only capable of driving loads at short distances. Long length wires will introduce excessive capacitive loading and cause the auto direction detection to fail. To overcome this drawback, the shield shield also has an optional on-board 74ABT125 buffer that is capable of driving heavier loads in *one* direction. A user can jumper wire to whichever IO pin they would like to be translated to 5V.

```

Comment:
No according to the [datasheets](https://github.com/spark/shields/tree/master/photon-shields/shield-shield/datasheets)